packages/core/src/App/Components/Elements/LiveChat/live-chat.tsx

```diff
@@ -1,41 +1,31 @@
 import { Icon, Popover, Text } from '@deriv/components';
 import { useIsLiveChatWidgetAvailable } from '@deriv/hooks';
-import { observer, useStore } from '@deriv/stores';
+import { observer } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
 import { Chat } from '@deriv/utils';
 import { useDevice } from '@deriv-com/ui';
 
-import useFreshChat from './use-freshchat';
-import useIntercom from './use-intercom';
+import useIsFreshchatAvailable from './useIsFreshchatAvailable';
+import useIsIntercomAvailable from './useIsIntercomAvailable';
```

Comment:
@nuzhy can 

`useIsFreshchatAvailable`
`useIsIntercomAvailable`

be part of the main hook? Do we really need to create a separate hook for them? 

```
import useFreshChat, {isFreshChatAvailable } from './use-freshchat';
import useIntercom, { isIntercomAvailable } from './use-intercom';

Reply:
can @prince-deriv, will update it